src/styles/style.scss

```diff
@@ -0,0 +1,142 @@
+
+$white: #fff;
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
+html {
+  font-family: "Roboto", sans-serif;
+  color: $main-accent;
+  box-sizing: border-box;
+}
+
+nav,
+ul,
+li,
+p,
+div,
+h3,
+article,
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
Something is wrong here - too many tags here

Are you sure that you
- really need to reset this set of styles for this set of elements?
- using effective selector?